features/step-definitions/draft_letter.steps.js

```diff
@@ -0,0 +1,37 @@
+const { When, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const SecurePage = require("../pageobjects/secure.page");
+const securePage = new SecurePage();
+const NewLetterPage = require("../pageobjects/new_letter.page");
+const newLetterPage = new NewLetterPage();
+const DraftsPage = require("../pageobjects/drafts.page");
+const draftsPage = new DraftsPage();
```

Comment:
no needs to use such exporting if you set this for your class
module.exports = new DraftsPage();
then you will need only 
const draftsPage = require("../pageobjects/drafts.page"); (name of constant - lowercase)